src/components/slider/slider.spec.js

```diff
@@ -1,5 +1,5 @@
-describe('md-slider', function() {
-  var $compile, $timeout, $log, $mdConstant, pageScope;
+describe('md-slider',function() {
+  var $compile,$timeout,$log,$mdConstant,pageScope;
```

Comment:
All of the spaces after commas in the file seem to have been removed. This causes a lot of unnecessary diffs. Please revert the removal of these spaces.

Reply:
Should be fixed